pixelegg/css/fancy.less

```diff
@@ -1,5 +1,5 @@
 /**
- * EGroupware: "fancy" theme
```

Comment:
Why do you change this?

Reply:
@ralfbecker , I reverted it now, sorry for this